File: src/main/java/com/moovie/backend/domain/model/MovieListDetail.java
package com.moovie.backend.domain.model;

import java.util.List;

public record MovieListDetail(
        int id,
        String name,
        String creator,
        String description,
        int movieCount,
        List<String> posterPaths,
        List<Movie> movies,
        int totalMovies,
        int totalPages,
        int commentsCount,
        int likesCount,
        boolean isLiked,
        List<String> tags
) {
}

File: src/main/kotlin/org/example/domain/model/MovieListDetail.kt
package org.example.domain.model

import kotlinx.serialization.Serializable

@Serializable
data class MovieListDetail(
    val id: Int,
    val name: String,
    val creator: String,
    val description: String? = null,
    val movies: List<Movie> = emptyList(),
    val totalMovies: Int = 0,
    val totalPages: Int = 0,
    val commentsCount: Int = 0,
    val likesCount: Int = 0,
    val isLiked: Boolean = false,
    val tags: List<String> = emptyList()
)
